lib/JitEngine.cpp

```diff
@@ -118,6 +118,8 @@ JitEngine::JitEngine() {
       getEnvOrDefaultBool("ENV_PROTEUS_SPECIALIZE_ARGS", true);
   Config.ENV_PROTEUS_SPECIALIZE_DIMS =
       getEnvOrDefaultBool("ENV_PROTEUS_SPECIALIZE_DIMS", true);
+  Config.ENV_PROTEUS_USE_HIP_RTC_CODEGEN =
+      getEnvOrDefaultBool("ENV_PROTEUS_USE_HIP_RTC_CODEGEN", true);
```

Comment:
Do we want to default to false?

Reply:
> Do we want to default to false?

I'm struggling to make a decision. HIP RTC might generate faster device code assuming the RTC optimization pipeline is better than our own, so I left on by default.

Reply:
Made my mind. Let's keep it as is, so default HIP RTC, until we're confident our own codegen is good. Configuration is an env var so easy to switch it a runtime without re-compilation.